lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1957,6 +1962,11 @@ SubExprInfo ExprEmitter::visitVerbatimExprOp(Operation *op, ArrayAttr symbols) {
   return {Unary, IsUnsigned};
 }
 
+SubExprInfo ExprEmitter::visitSV(MacroRefExprOp op) {
+  os << "`" << op.ident().getName();
+  return {Symbol, IsUnsigned};
```

Comment:
how do we know the expression is unsigned?  It also almost certainly isn't a Symbol precedence.  You might need to make a new precedence level.  Since it may expand by the preprocessor into any expression, unless it's a rhs of an assignment, I think you need to mark it so it will get parenthesis.